cmd/internal/shared/events.go

```diff
@@ -27,15 +28,15 @@ import (
 )
 
 // InitializeEvents initializes the event system.
-func InitializeEvents(ctx context.Context, config config.ServiceBase) (err error) {
+func InitializeEvents(ctx context.Context, c *component.Component, config config.ServiceBase) error {
```

Comment:
nit; not a fan of one-letter name here, since all three parameter types start with `c`, IMO this is quite confusing. Consider `comp` ?
While we're at it, `config` should be called `conf` https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.9/DEVELOPMENT.md#single-word-entities
So maybe refactor these altogether?

Reply:
The catch is that our whole codebase uses `c` as the common name for a `*component.Component`. While I can go for `comp`, it will be the odd one out. If we decide to change the abbreviation, that's fine by me but I'd rather fix it everywhere (and not as part of this PR, since it's out of scope).